wsgidav/dav_provider.py

```diff
@@ -850,9 +850,9 @@ def is_locked(self):
             return False
         return self.provider.lock_manager.is_url_locked(self.get_ref_url())
 
-    def remove_all_locks(self, *, recursive):
+    def remove_all_locks(self, *, recursive=False):
```

Comment:
I'd prefer, not to introduce a default. The current idea is to force the caller to make an explicit statement.